x-pack/solutions/security/plugins/security_solution/public/common/components/security_route_page_wrapper/index.tsx

```diff
@@ -91,7 +94,7 @@ export const SecurityRoutePageWrapper: FC<PropsWithChildren<SecurityRoutePageWra
   return (
     <TrackApplicationView viewId={pageName}>
       {children}
-      <SpyRoute pageName={pageName} />
+      {!omitSpyRoute && <SpyRoute pageName={pageName} />}
```

Comment:
This is great 👏 
Would it be possible to add this test case for this prop in the test for this component that already exists?